django_adobesign/exceptions.py

```diff
@@ -1,44 +1,64 @@
 """Custom exceptions."""
-from django.utils import six
 
-from requests.exceptions import RequestException
 
-
-class AdobeSignException(Exception):
-    """An error occurred with AdobeSign wrapper API."""
+class AdobeExceptionFactory(object):
```

Comment:
Why `get_exception` is not simply a method ? I thing we don't need a class wrapper

Reply:
ok